src/suspectplacesearcher/main.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+from geopy.distance import vincenty
+import requests
+import math
+
+
+class SuspectPlaceSearch(object):
```

Comment:
I don't prefer adopted this name cause SexRelated is too much strict,  being the Class construct to be expansible to search for any kind of company we judge to be Suspect being sexual or not.  Just incrementing the Keyword vector.